packages/app/src/cli/models/app/app.test-data.ts

```diff
@@ -202,6 +202,22 @@ export const testRemoteSpecifications: RemoteSpecification[] = [
       },
     },
   },
+  {
+    name: 'App Setup Extension',
```

Comment:
I think it is a good idea that this matches the app_config extension we are trying to build. 



Here is the PR for reference https://github.com/Shopify/shopify/pull/415088

Reply:
That makes sense. Let me change it.